test/routes/items_spec.js

```diff
@@ -0,0 +1,99 @@
+const expect = require( 'chai' ).expect
+const sinon = require( 'sinon' )
+const express = require( 'express' )
+
+// const stubbedResponse = ( expected, done ) => {
+//   return {
+//     render: data => {
+//       expect( data ).to.be.eql( expected )
+//       done()
+//     }
+//   }
+// }
+
+describe( 'items routes', () => {
+  let sandbox
+  let router
+
+  beforeEach( () => {
+    sandbox = sinon.sandbox.create()
+    sandbox.stub( express, 'Router' ).returns({
+      get: sandbox.spy(),
+      post: sandbox.spy(),
+      delete: sandbox.spy()
+    })
+
+    router = require( '../../routes/items' )
+  })
+
+  it( 'registers URI / for GET', () => {
+    expect( router.get.calledWith( '/', sandbox.match.any )).to.be.true
+  })
+
+  describe( 'GET /starred URI', () => {
+    it( 'is registered', () => {
+      expect( router.get.calledWith( '/starred', sandbox.match.any )).to.be.true
+    })
+
+    it( 'renders the things', done => {
+      const callback = router.get.secondCall.args[ 1 ]
+      const STUBBED_DATA = { hi: 'world' }
+
+      const request = {
+        app: {
+          get: data => ({ Item: {
+            findAll: () => new Promise( (resolve, reject) => resolve( STUBBED_DATA ) )
+          } })
+        },
+        user: { id: 1, query: 'query' }
+      }
+
+      const response = {
+        json: data => {
+          expect( data ).to.eql({ data: STUBBED_DATA })
+          done()
+        }
+      }
+
+      callback( request, response )
+    })
+  })
+
+  describe( 'POST / URI', () => {
+    it( 'is registered', () => {
+      expect( router.post.calledWith( '/', sandbox.match.any )).to.be.true
+    })
+
+    it( 'responds with a redirect to /items', done => {
+      const callback = router.post.firstCall.args[ 1 ]
+      const STUBBED_DATA = { hi: 'world' }
+      const stubbedResponse = ( expected, done ) => {
+        return {
+          redirect: url => {
+            expect( url ).to.be.eql( expected )
+            done()
+          }
+        }
+      }
+      const request = {
+        app: {
+          get: data => ({ Item: {
+            create: () => new Promise( (resolve, reject) => resolve( STUBBED_DATA ) )
+          } })
+        },
+        body: {
```

Comment:
`body` and `user` entries contain data that is unused in this test, which makes the test a little bit harder to parse for me.  Since it's unused, I would consider moving it outside of the test block so that this code focuses on the actual expectation, rather than the extraneous setup details:
```js
// Probably at the top of this file?
const REQUEST_DATA = {
  body: { 
    title: 'title',
    description: 'description',
    parent_id: 1
  },
  user: { id: 1, query: 'query' }
}
```
Now, the test can be a little more concise in its setup:
```js
const request = {
  app: { /* app stuff */
}

calback( Object.assign( {}, REQUEST_DATA, request ), response )
```